### PR TITLE
feat(shutdown):  Adds option to exit(o) instead of resending the uncatched signal that was received

### PR DESCRIPTION
**Pull request by aeimer**
Superseeds #211 

FYI @gergelyke 

Adds option to exit(o) instead of resending the uncatched signal that was received.

### CODE REVIEW

File: lib/terminus.spec.js [hunk 1]
```diff
@@ -658,25 +658,37 @@ describe('Terminus', () => {
...
-
-  const res = await fetch('http://localhost:8000/health')
-  expect(res.status).to.eql(200)
-  expect(res.headers.has('Access-Control-Allow-Methods')).to.eql(true)
-  expect(res.headers.get('Access-Control-Allow-Methods')).to.eql('OPTIONS, POST, GET')
-  expect(res.headers.has('Access-Control-Allow-Origin')).to.eql(true)
-  expect(res.headers.get('Access-Control-Allow-Origin')).to.eql('*')
+  .then(res => {
+    // FIXME: Check if exit code was 0
+    // expect()
```
Comment (gergelyke) [inline lib/terminus.spec.js:692]: can you fix the ident here please? and also add the expectation that you started on line 692?

Comment (aeimer) [reply lib/terminus.spec.js:692]: @gergelyke actually I asked you for help in the previous PR. So maybe you can give me a hint on how to get the exit code from the execFile function?

Comment (aeimer) [reply lib/terminus.spec.js]: Ok fixed

**Comment by gergelyke** [comment]
thank you @aeimer 🙇

**Pull request merged**